src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -62,6 +62,7 @@ import {
   removeSelectedAnswer,
   unpopulatePrepopulatedAnswers,
   prepopulateAnswers,
+  unretireMapping,
```

Comment:
How is this related to the ticket?

Reply:
It was affecting my coverage but I have now created a separate [PR](https://github.com/openmrs/openmrs-ocl-client/pull/382) for it.